app/serializers.py

```diff
@@ -0,0 +1,28 @@
+import json
+from xml.etree import ElementTree
+
+from app.book_model import Book
+
+
+class BookSerializerJson:
+
+    @staticmethod
+    def serialize_json(book: Book, serialize_type: str) -> str:
+        if serialize_type == "json":
+            return json.dumps({"title": book.title, "content": book.content})
+        else:
+            raise ValueError(f"Unknown serialize type: {serialize_type}")
+
+
+class BookSerializerXml:
+    @staticmethod
+    def serialize_xml(book: Book, serialize_type: str) -> str:
```

Comment:
Create a separate `Serializer` abstract class with `serialize`  method. Define two subclasses for XML and JSON serializers. Define common `serialize` method that would be called from these serializers. After that, create a `BookSerializer` that would define which serializer is called. Apply similar changes to the other places as well